.github/workflows/integration_test.yml

```diff
@@ -1,8 +1,25 @@
 name: Integration Tests
 on: push
 jobs:
+  check-secret:
```

Comment:
I'd like to nitpick on naming. Let's call this job `check-slack-api-token` and set `SLACK_API_TOKEN_EXISTS`?

Reply:
Updated naming.